mathics/core/pattern.py

```diff
@@ -33,6 +34,11 @@
 
 
 def Pattern_create(expr):
```

Comment:
Again, I realize this is legacy stuff, but we shouldn't have functions that start with capital letters. 

Suggested replacement signature: 

```
def pattern_create(expr: BaseElement) -> Union[AtomPattern | ExpressionPattern]:
       """
       If ``expr`` is listed in ``pattern_object``  return the pattern found there. 
       Otherwise, if ``expr`` is an ``Atom``, create and return  ``AtomPattern`` for ``expr``.
       Otherwise, create and return and ``ExpressionPattern`` for ``expr``.
       """

Reply:
Also, do you know why we do on line 86:

```
   create = staticmethod(Pattern_create)
```

rather than use:

```
    @staticmethod
    def pattern_create(...)
``` 
?

Reply:
#613 attempts to do this. This is a holdover from the Jan days.